common/proxy/proxy_factory/default.go

```diff
@@ -51,6 +61,86 @@ func (factory *DefaultProxyFactory) GetProxy(invoker protocol.Invoker, url *comm
 	return proxy.NewProxy(invoker, nil, attachments)
 }
 func (factory *DefaultProxyFactory) GetInvoker(url common.URL) protocol.Invoker {
-	// todo: call service
-	return protocol.NewBaseInvoker(url)
+	return &ProxyInvoker{
+		BaseInvoker: *protocol.NewBaseInvoker(url),
+	}
+}
+
+type ProxyInvoker struct {
+	protocol.BaseInvoker
+}
+
+func (pi *ProxyInvoker) Invoke(invocation protocol.Invocation) protocol.Result {
+	result := &protocol.RPCResult{}
+	result.SetAttachments(invocation.Attachments())
+
+	url := pi.GetUrl()
+
+	methodName := invocation.MethodName()
+	proto := url.Protocol
+	path := strings.TrimPrefix(url.Path, "/")
+	args := invocation.Arguments()
+
+	// get service
+	svc := common.ServiceMap.GetService(proto, path)
+	if svc == nil {
+		logger.Errorf("cannot find service [%s] in %s", path, proto)
```

Comment:
add "[ProxyInvoker][Invoke] as log prefix.

Reply:
logger will show the stack